inc/classes/class-srm-post-type.php

```diff
@@ -746,6 +801,7 @@ public function load_resources() {
 		if ( 'redirect_rule' === get_post_type() ) {
 			wp_enqueue_style( 'redirectjs', plugin_dir_url( 'safe-redirect-manager/safe-redirect-manager.php' ) . 'assets/css/redirect.css', array(), SRM_VERSION );
 			wp_enqueue_script( 'redirectjs', plugin_dir_url( 'safe-redirect-manager/safe-redirect-manager.php' ) . 'assets/js/redirect.js', array( 'jquery' ), SRM_VERSION );
+			wp_enqueue_script( 'quick-bulk-editjs', plugin_dir_url( 'safe-redirect-manager/safe-redirect-manager.php' ) . 'assets/js/quick-bulk-edit.js', array( 'jquery', 'inline-edit-post' ), SRM_VERSION );
```

Comment:
@dhanendran I suggest not using a hard-coded plugin file name. It can possibly break if the plugin directory name does not match.

cc: @faisal-alvi @dkotter